packages/tools/devtools/devtools-core/package.json

```diff
@@ -118,6 +118,58 @@
 		}
 	},
 	"typeValidation": {
-		"broken": {}
+		"broken": {
```

Comment:
Just a heads-up, I'm making some breaking changes to our enums, which will conflict with these changes and your additional `EditType` flags.

#16744

Reply:
The above PR just got merged, so you'll have some merge conflicts to resolve here. I think you should be able to just resolve these using the contents from main, and everything should just work (and you'll have a smaller PR diff).